tests/tga/bug00247.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+
+#include "gd.h"
+#include "gdtest.h"
+
+int main(int argc, char **argv)
```

Comment:
I get compile time warnings wrt. to `argc` and `argv` being unused. IIRC, that causes the build to fail in the GD-2.2 branch (or did it use to fail in master, and this has recently been changed?)

Otherwise the PR looks fine. Thanks!